### DISCUSSION TITLE
Why am I getting prompted multiple times when importing on Linux?

### DISCUSSION THREAD

**Discussion by RobertBernstein** [beetbox/beets#4673] [Q&A] [answered]
tl;dr - I have used beets on Windows for years.  When I run `beet import .`, it searches for the matching album on MusicBrainz and then imports all of the songs in one step.  I've moved my beets install to a docker container running Arch (?) Linux and when I run `beet import *`, it prompts me for each song in the directory that's part of the same album instead of recognizing that they are all part of one album.  Any idea why it does this?

------------------------------------------------------------------------------------------------------------------------

**More detail:**

For example, I have all the songs from a 2 CD set in a single directory, but get the following output:

```
root@beets:/downloads# beet import *
No files imported from /downloads/@eaDir
No files imported from /downloads/Thumbs.db

/downloads/Yes - Featuring Anderson, Rabin & Wakeman - Live At The Apollo - Disc 1 of 2 - 01 - Intro  Cinema  Perpetual Change.mp3 (1 items)
Finding tags for album "Yes - Featuring Anderson, Rabin & Wakeman - Live At The Apollo - Disc 1 of 2".
Candidates:
1. Yes featuring Jon Anderson, Trevor Rabin, Rick Wakeman - 50th Anniversary: Live at the Apollo (38.3%) (missing tracks, album, artist) (2xCD, 2018, XE, Eagle Records, EDGCD676)
2. Yes featuring Jon Anderson, Trevor Rabin, Rick Wakeman - 50th Anniversary: Live at the Apollo (33.2%) (missing tracks, album, mediums, ...) (Digital Media, 2018, XW, Mercury Studios)
3. Daryl Hall & John Oates - Live at the Apollo (23.7%) (missing tracks, artist, tracks, ...) (Blu-spec CD, 2011, JP, RCA, SICP-20281, Remastered)
4. Jon Anderson/Rick Wakeman - The Living Tree Live (16.1%) (missing tracks, album, tracks, ...) (CD, 2012, XW, Gonzo! Records, HST097CD)
# selection (default 1), Skip, Use as-is, as Tracks, Group albums,
Enter search, enter Id, aBort? 1
Correcting tags from:
    Yes - Featuring Anderson, Rabin & Wakeman - Live At The Apollo - Disc 1 of 2
To:
    Yes featuring Jon Anderson, Trevor Rabin, Rick Wakeman - 50th Anniversary: Live at the Apollo
URL:
    https://musicbrainz.org/release/83238c6e-c02c-4239-a4c1-e4f593404c68
(Similarity: 38.3%) (missing tracks, album, artist) (2xCD, 2018, XE, Eagle Records, EDGCD676)
CD 1
Missing tracks (11/12 - 91.7%):
 ! Hold On                                                                                    (# 2) (8:00)
 ! I’ve Seen All Good People: (i) Your Move (ii) All Good People                              (# 3) (7:43)
 ! Lift Me Up                                                                                 (# 4) (7:11)
 ! And You & I: (i) Cord of Life (ii) Eclipse (iii) The Preacher, The Teacher (iv) Apocalypse (# 5) (10:40)
 ! Rhythm of Love                                                                             (# 6) (4:54)
 ! Heart of the Sunrise                                                                       (# 7) (11:26)
 ! Changes                                                                                    (# 8) (6:59)
 ! Long Distance Runaround / The Fish (Schindleria Praematurus)                               (# 9) (6:17)
 ! Orchestral Shade / Awaken                                                                  (#10) (22:40)
 ! Make It Easy / Owner of a Lonely Heart                                                     (#11) (9:46)
 ! Roundabout                                                                                 (#12) (7:34)
Apply, More candidates, Skip, Use as-is, as Tracks, Group albums,
Enter search, enter Id, aBort? a
```

On Windows, I'm pretty sure it would have imported the entire collection.  Why is beet on Linux now looking at track 2 and prompting me for which album it is all over again?

```
/downloads/Yes - Featuring Anderson, Rabin & Wakeman - Live At The Apollo - Disc 1 of 2 - 02 - Hold On.mp3 (1 items)
Finding tags for album "Yes - Featuring Anderson, Rabin & Wakeman - Live At The Apollo - Disc 1 of 2".
Candidates:
1. Yes featuring Jon Anderson, Trevor Rabin, Rick Wakeman - 50th Anniversary: Live at the Apollo (38.3%) (missing tracks, album, artist) (2xCD, 2018, XE, Eagle Records, EDGCD676)
2. Yes featuring Jon Anderson, Trevor Rabin, Rick Wakeman - 50th Anniversary: Live at the Apollo (33.2%) (missing tracks, album, mediums, ...) (Digital Media, 2018, XW, Mercury Studios)
3. Daryl Hall & John Oates - Live at the Apollo (24.7%) (missing tracks, artist, tracks, ...) (Blu-spec CD, 2011, JP, RCA, SICP-20281, Remastered)
4. Jon Anderson/Rick Wakeman - The Living Tree Live (15.6%) (missing tracks, album, tracks, ...) (CD, 2012, XW, Gonzo! Records, HST097CD)
# selection (default 1), Skip, Use as-is, as Tracks, Group albums,
Enter search, enter Id, aBort? 1
Correcting tags from:
    Yes - Featuring Anderson, Rabin & Wakeman - Live At The Apollo - Disc 1 of 2
To:
    Yes featuring Jon Anderson, Trevor Rabin, Rick Wakeman - 50th Anniversary: Live at the Apollo
URL:
    https://musicbrainz.org/release/83238c6e-c02c-4239-a4c1-e4f593404c68
(Similarity: 38.3%) (missing tracks, album, artist) (2xCD, 2018, XE, Eagle Records, EDGCD676)
CD 1
Missing tracks (11/12 - 91.7%):
 ! Intro / Cinema / Perpetual Change                                                          (# 1) (11:20)
 ! I’ve Seen All Good People: (i) Your Move (ii) All Good People                              (# 3) (7:43)
 ! Lift Me Up                                                                                 (# 4) (7:11)
 ! And You & I: (i) Cord of Life (ii) Eclipse (iii) The Preacher, The Teacher (iv) Apocalypse (# 5) (10:40)
 ! Rhythm of Love                                                                             (# 6) (4:54)
 ! Heart of the Sunrise                                                                       (# 7) (11:26)
 ! Changes                                                                                    (# 8) (6:59)
 ! Long Distance Runaround / The Fish (Schindleria Praematurus)                               (# 9) (6:17)
 ! Orchestral Shade / Awaken                                                                  (#10) (22:40)
 ! Make It Easy / Owner of a Lonely Heart                                                     (#11) (9:46)
 ! Roundabout                                                                                 (#12) (7:34)
Apply, More candidates, Skip, Use as-is, as Tracks, Group albums,
Enter search, enter Id, aBort? a
This album is already in the library!
Old: 1 items, MP3, 278kbps, 11:20, 22.8 MiB
New: 1 items, MP3, 296kbps, 8:00, 17.1 MiB
[S]kip new, Keep all, Remove old, Merge all? m
```

**Comment by wisp3rwind** [comment, marked as answer]
Because you're telling it to: `beet import *` is expanded by the shell to `beet import <file1> <file2> ...`, i.e. each file is passed as separate argument. Why did you switch from `beet import .` to `beet import *`?

**Reply by RobertBernstein** [reply]
I think I tried `beet import .` and it didn't work. I'll try it again.

**Reply by RobertBernstein** [reply]
Thanks, @wisp3rwind!  That worked.  I'm good now.
